### PR TITLE
removed Directory.Read.All from the list of perms for List domainNameReferences

### PR DESCRIPTION
**Pull request by adimitui**

### CODE REVIEW

**Comment by adimitui** [comment]
#sign-off

**Comment by adimitui** [comment]
[\@davidmu1](https://github.com/davidmu1) **Signed-off.**

**Comment by eketo-msft** [comment]
@adimitui, https://docs.microsoft.com/en-us/graph/permissions-reference doesn't have Domain listed at all?  Can we fix that at the same time?

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -459,6 +459,15 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
+## Domain permissions
```
Comment (Lauragra) [inline concepts/permissions-reference.md]: If this is a new permission, we should have an entry in the changelog calling this out as well.

Comment (eketo-msft) [reply concepts/permissions-reference.md]: It is not new, the resource doc has mentioned it for a long time but this specific permissions reference doc was missed.

Comment (Lauragra) [reply concepts/permissions-reference.md]: Okay, great. Approving.

**Pull request merged**